skorch/tests/callbacks/test_logging.py

```diff
@@ -10,7 +10,7 @@ class TestPrintLog:
     @pytest.fixture
     def print_log_cls(self):
         from skorch.callbacks import PrintLog
-        keys_ignored = ['batches', 'dur', 'text']
+        keys_ignored = ['batches', 'dur', 'text', 'event_odd']
```

Comment:
'batches' is not necessary anymore

Reply:
Right, I was actually suggesting this in the [other thread](https://github.com/dnouri/skorch/pull/288#issuecomment-405922282).

What is this 'text' by the way? Should I remove it also?

Reply:
Yeah you can remove it as well, seems to be an artifact from the past.